### PR TITLE
 Avatar 

### PR DESCRIPTION
**Pull request by khushijohri2001**
Please review my code for my Component Library's Avatar Component.

The page includes:

- Header with Girlifycoder Logo, Heading, and Github link
- The navigation bar on the left
- Sidebar on the right
- Details about the Component and snippet

https://user-images.githubusercontent.com/65448358/160250696-2cf2f179-f18c-4fd5-a161-88fbc0c8c4e5.mp4

### CODE REVIEW

File: avatars/avatars.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  <head>
+    <title>Avatars</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="../home/styles.css" />
+    <link rel="stylesheet" href="./styles2-avatars.css" />
+  </head>
+
+  <body>
+    <div class="main-container">
+      <!-- Header -->
```
Comment (utkarshducoder) [inline avatars/avatars.html]: remove the comment from here

File: avatars/avatars.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        <div class="header-title"><h1>GirlifyCoder Component Library</h1></div>
+        <div class="header-icons">
+          <a href="https://github.com/khushijohri2001"
+            ><img src="../images/github.svg"
+          /></a>
+          <small>Source Code</small>
+        </div>
+      </header>
+
+      <!-- Left-Aside -->
```
Comment (utkarshducoder) [inline avatars/avatars.html]: here also

**Review by utkarshducoder — APPROVED** [review]

File: avatars/styles2-avatars.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+.avatars-sub {
+  margin: 1rem 0rem;
+  color: white;
+  font-size: 1.2rem;
+}
+.avatars-snippet {
+  margin: 1rem 0rem;
+}
+.standard-avatars {
+  color: yellow;
```
Comment (pnchinmay) [inline avatars/styles2-avatars.css]: Hex code recommended

**Review by pnchinmay — APPROVED** [review]
Left a few comments
Everything else looks good :100:

File: avatars/avatars.html (hunk 1 above)
Comment (khushijohri2001) [reply avatars/avatars.html]: It is a required comment for reviewers to understand better

File: avatars/styles2-avatars.css (hunk 3 above)
Comment (khushijohri2001) [reply avatars/styles2-avatars.css]: done

**Pull request merged**